runtime/src/lib.rs

```diff
@@ -394,11 +395,11 @@ parameter_types! {
     pub HistoryDepth: u32 = 30; // Number of eras to keep in history. Older eras cannot be claimed.
 }
 
-pub const MAX_AUTHORITIES: u32 = 10;
-pub const MAX_VOTERS: u32 = 10;
+pub const MAX_AUTHORITIES: u32 = 32;
```

Comment:
Duplication here with MaxActiveValidators ?

Reply:
Not quite, `MAX_AUTHORITIES` accounts for all validators, that is active + eligible, whereas `MaxActiveValidators` accounts only for the subset of active validators.
They are different things, so I'd rather keep them separated in the code, even if in the current configuration we use the same value for both.

Reply:
I have renamed `MAX_AUTHORITIES` and added some comments, I hope it's more intuitive now